app/assets/stylesheets/_state-file.scss

```diff
@@ -748,6 +748,13 @@ $state-colors: (
   }
 }
 
+.federal-info-outer {
+  .dev-button {
+    margin: 0.5rem auto;
+    display: block;
+  }
+}
+
```

Comment:
I added this style here so that the "Continue Skipping schema validation" button would not get it's text truncated:
![image](https://github.com/user-attachments/assets/1c94cc32-8cea-4283-b526-f3b4115e6c60)